### PR TITLE
OKTA-302725: Add okta.oauth2.postLogoutRedirectUri to Readme

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**
Issue:

https://oktainc.atlassian.net/browse/OKTA-302725

### CODE REVIEW

File: okta-hosted-login/README.md [hunk 1]
```diff
@@ -32,7 +32,8 @@ Plug these values into the `mvn` commands used to start the application.
 cd okta-hosted-login
 mvn -Dokta.oauth2.issuer=https://{yourOktaDomain}/oauth2/default \
     -Dokta.oauth2.clientId={clientId} \
-    -Dokta.oauth2.clientSecret={clientSecret}
+    -Dokta.oauth2.clientSecret={clientSecret} \
+    -Dokta.oauth2.postLogoutRedirectUri={logoutRedirectUri}
```
Comment (bdemers) [inline okta-hosted-login/README.md:36]:
```suggestion
    -Dokta.oauth2.postLogoutRedirectUri={absoluteLogoutRedirectUri} # (optional) add description of when this should be used
```

File: okta-hosted-login/README.md [hunk 2]
```diff
@@ -32,7 +32,8 @@ Plug these values into the `mvn` commands used to start the application.
 cd okta-hosted-login
 mvn -Dokta.oauth2.issuer=https://{yourOktaDomain}/oauth2/default \
     -Dokta.oauth2.clientId={clientId} \
-    -Dokta.oauth2.clientSecret={clientSecret}
+    -Dokta.oauth2.clientSecret={clientSecret} \
+    -Dokta.oauth2.postLogoutRedirectUri={absoluteLogoutRedirectUri} # (optional) add description of when this should be used
```
Comment (bdemers) [inline okta-hosted-login/README.md:36]: ^ we should add an actual description ;)

Comment (arvindkrishnakumar-okta) [reply okta-hosted-login/README.md:36]: Ah, sorry.. will include one.

File: okta-hosted-login/README.md [hunk 3]
```diff
@@ -32,7 +32,8 @@ Plug these values into the `mvn` commands used to start the application.
 cd okta-hosted-login
 mvn -Dokta.oauth2.issuer=https://{yourOktaDomain}/oauth2/default \
     -Dokta.oauth2.clientId={clientId} \
-    -Dokta.oauth2.clientSecret={clientSecret}
+    -Dokta.oauth2.clientSecret={clientSecret} \
+    -Dokta.oauth2.postLogoutRedirectUri={absoluteLogoutRedirectUri} # (optional) add this when you want users to be logged out of the app (session cleared) and redirected to a uri.
```
Comment (bdemers) [inline okta-hosted-login/README.md:36]: This should say something about either an SSO logout or the OAuth term RP-Initiated logout
(Without this option the session of the application will still be cleared).

Maybe something like:
> (optional) configure this property to enable SSO logout
or :
> (optional) configure this property to enable an RP-initiated logout

(my concern is most folks will no know what an RP initiated logout is, so SSO logout might be a better fit 🤔

Comment (arvindkrishnakumar-okta) [reply okta-hosted-login/README.md:36]: ok noted. Will update to `SSO logout`.

Comment (arvindkrishnakumar-okta) [reply okta-hosted-login/README.md]: done

**Pull request merged**